src/src/Helpers/Misc.cpp

```diff
@@ -224,6 +224,18 @@ String getTaskValueName(taskIndex_t TaskIndex, uint8_t TaskValueIndex) {
   return ExtraTaskSettings.TaskDeviceValueNames[TaskValueIndex];
 }
 
+/*********************************************************************************************
+ * get the taskPluginID with required checks, INVALID_PLUGIN_ID when invalid
+ ********************************************************************************************/
+pluginID_t getTaskDevicePluginID(taskIndex_t TaskIndex) {
```

Comment:
We don't have a `getPluginIDfromTaskIndex` function?

I think it should be implemented in Globals/Plugins.h / .cpp

Reply:
I will rename and move it there 👍